inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -40,6 +40,7 @@
 
 
 INSPIRE_BAI = re.compile('(\w+\.)+\d+')
+LOOKS_LIKE_CERN = re.compile('^\d+$|^CER[MN]|^CNER-|^CVERN', re.I)
```

Comment:
...and I'm not even done! Look at this beauty:
```XML
<datafield tag="035" ind1=" " ind2=" ">
  <subfield code="9">INSPIRE</subfield>
  <subfield code="a">CER-749663</subfield>
</datafield>
```
(https://inspirehep.net/record/1068076/export/xme)